.github/workflows/build-instructions.yml

```diff
@@ -20,16 +20,25 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
-    - name: Build, install and test (Docker)
+    - name: Build and install (Docker)
       # Note: this _must_ be kept in sync with README.md until we
       # have an established mechanism to test doc-included code.
       run: |
-        docker run -v `pwd`:/src aswf/ci-base:2022.2 bash -c '
+        docker run -v `pwd`:/src ghcr.io/openassetio/openassetio-build bash -c '
           cd /src && \
           cmake -S . -B build && \
           cmake --build build && \
           cmake --install build'
 
+    - name: Build, install and test (Docker)
+      # Note: this _must_ be kept in sync with README.md until we
```

Comment:
[consider][minor] Rather than duplicating this comment, we could move it one level up to the job level, or even the whole workflow level.

Reply:
Good idea c4c6231033860c9dd933a144acc33d7d8622a7f5